css/home.css

```diff
@@ -98,25 +98,25 @@
     position: relative;
   }
 
-  .footer--text {
+  .footer__text {
     word-break: break-word;
     text-align: center;
     font-size: 16px;
     color:white;
     font-family: Roboto;
   }
 
-  .footer-icons:nth-last-child(1) {
+  .footer__icons:nth-last-child(1) {
     margin: 0;
     display: inline-block;
     padding:34px;
   }
 
-  .footer--hover{
+  .footer__hover{
```

Comment:
please avoid repeating blocks of code, and also check slack as i will post an explanation on cleaner media query handling.